### DISCUSSION TITLE
error when pulling from latest to dev

### DISCUSSION THREAD

**Discussion by loulou91** [photoprism/photoprism#3552] [Q&A]
Hello, I wanted to test dev version, so I modified docker-compose.yml "image: photoprism/photoprism" from  latest to dev.
When pulling I got the following error:
![image](https://github.com/photoprism/photoprism/assets/51127123/123ebf97-dfbd-4f8e-b4cc-6f534286a44f)
Any idea?

NUC i5, Proxmox 7.4, MV debian 11.6, docker 24.0.2, Docker Compose v2.18.1

**Comment by graciousgrey** [comment]
Hey :)

If you want to test our development preview you need to pull photoprism/photoprism:preview instead of photoprism/photoprism:latest. 
photoprism/photoprism:dev does not exist.

**Comment by loulou91** [comment]
Of course! sorry for disturbance!
